### ISSUE TITLE
Applying outside of review

### ISSUE THREAD

**Issue by bcbwatt** [eshapard/experimentalCardEaseFactor#8]
I have this installed, but I do most of my reviews on mobile (so this addon does not apply).

Is is possible to adjust the ease factor for all cards, outside of the review period? Perhaps with a menu option to run through the decks.

**Comment by filipenanclarez** [comment]
Yeah, this is a great idea, can you do it, with the similar code in in https://github.com/eshapard/Anki-avgEase ?

If you can loop through deck options, i think you also can loop trough deck cards, and update the ease factor.

Then mobile users can periodicly do an ease factor update on desktop.

> I have this installed, but I do most of my reviews on mobile (so this addon does not apply).
> 
> Is is possible to adjust the ease factor for all cards, outside of the review period? Perhaps with a menu option to run through the decks.

**Comment by eshapard** [comment]
Sorry, I don't use Anki anymore, so I have no motivation to work on 
these scripts.

Excerpts from filipenanclarez's message of February 3, 2020 11:47 am:

**Comment by Willz7** [comment]
@eshapard What made you stop using Anki?

**Comment by galantra** [comment]
Do you use an alternative?

**Comment by eshapard** [comment]
I use [Ebisu](github.com/fasiha/ebisu) to model the recall probability 
of items over time and created my own flashcard application to handle 
everything else. That gives me a lot of flexibility to implement some 
features that Anki doesn't have, and Ebisu let's me select items based 
on probability of recall rather than working around the 
schedule-a-review-date paradigm that Anki (and most other flashcard 
programs) inherited from Supermemo.

Some day I might write about what I'm doing, and maybe publish some 
code, but that won't be for a while. I suspect that what I'm doing won't 
have the kind of mass appeal that Anki has, as I mostly use a simple 
text-based (curses) interface.

If you're interested, and know some python, I did publish some of the 
code I use for creating nested and overlapping cloze deletions somewhere 
on eshapard.github.io.

Excerpts from galantra's message of April 25, 2020 7:13 am:
